test/e2e/go.mod

```diff
@@ -0,0 +1,37 @@
+module github.com/ovn-org/ovn-kubernetes/test/e2e
+
+go 1.13
```

Comment:
@as-com I think we only require Go 1.12; either we should only require 1.12 here or we should bump master to 1.13